src/bmeg/utils.py

```diff
@@ -16,6 +16,9 @@ def check_types(*args, **kwargs):
         parameters = dict(zip(spec.args, args))
         parameters.update(kwargs)
         for name, value in parameters.items():
+            # do not check null parameters
+            if value is None:
+                continue
```

Comment:
We should be careful here.

For example, in the case of a field `my_extras: dict`, this would allow the value to `None. JSON will serialize `None` as `null. So now, some objects/vertices have a dictionary, and some have `null`.

I think we should keep field types as consistent as possible. For example `my_extras` should always be a dictionary, even if it's empty.